FirebaseAuth/Sources/Swift/Backend/RPC/RevokeTokenRequest.swift

```diff
@@ -80,7 +80,7 @@ private let kIDTokenKey = "idToken"
   }
 
   @objc public init(withToken token: String,
-                    idToken: String,
+                    idToken: String?,
```

Comment:
idToken is actually required for RevokeTokenRequest.

Reply:
Restored back to non-optional - which means the ObjC SDK and unit test is wrong.

Reply:
Fixed in https://github.com/firebase/firebase-ios-sdk/pull/11419